code/game/machinery/cryopod.dm

```diff
@@ -308,6 +310,17 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 					mob_occupant.transferItemToLoc(W, control_computer, TRUE)
 				else
 					mob_occupant.transferItemToLoc(W, loc, TRUE)
+		if(istype(W, /obj/item/clothing))
+			var/obj/item/clothing/cloth = W
+			if(cloth.tracking_bug)
+				cloth.tracking_bug.forceMove(drop_location())
+				if(control_computer && control_computer.allow_items)
+					control_computer.frozen_items += W
+					mob_occupant.transferItemToLoc(cloth.tracking_bug, control_computer, TRUE)
+				else
+					mob_occupant.transferItemToLoc(cloth.tracking_bug, loc, TRUE)
+				cloth.tracking_bug.detach_from_clothing(mob_occupant, cloth)
```

Comment:
This seems way too specific, is there a better way of doing this that doesnt require specificly type checking for the bug when they cryo?